core/src/runtime/mod.rs

```diff
@@ -320,16 +349,29 @@ impl Runtime {
     /// Installs a periodic pipeline to a core.
     ///
     /// `core` is the logical id that identifies the core. The `installer`
-    /// is a closure that takes in a hashmap of `PortQueue`s and returns a
-    /// `Pipeline` that will be run periodically every `dur` interval.
+    /// is a closure that takes in a hashmap of [`PortQueue`]`s
+    /// and returns a [`Pipeline`] that will be run
+    /// periodically every `dur` interval.
```

Comment:
formatting here is wacky.

Reply:
there's an extra ` after the first [PortQueue] that's screwing up the formatting in this section.

Reply:
should be a `'`

Reply:
actions no symbol. fixed.